w3c-ruleset-review-process.md

```diff
@@ -0,0 +1,88 @@
+# ACT Review Process
+
+This ACT Rules Format is intended to enable a consistent interpretation of how to test conformance to WCAG and other accessibility requirements documents. The W3C is looking to establish a commonly agreed upon set of ACT Rules, to help organisations across the field of web accessibility to produce consistent test results. This common set is the WCAG Ruleset, which is maintained by the ACT Task Force.
+
+Organisations looking to contribute rules to the WCAG Ruleset can submit rules to the ACT Task Force. The ACT Task Force will review any rules that have been proposed, and decided whether or not it can be included in WCAG Ruleset.
+
+When a rule is submitted to the ACT Task Force, the Task Force expects for that rule to be as complete and as accurate as the rule author can possibly make it. Organisations creating rules are expected to have their own review and quality assurance processes in place. 
+
+## Proposing And Updating A Rule
+
+When proposing a new rule, or an update to a rule, there are four steps to go through:
+
+1. **Proposal**: The rule author creates an issue
+2. **Screening**: The ACT Task Force facilitators do an initial review
+3. **Survey**: The ACT Task Force reviews the rule through a survey
+5. **Publication**: The ACT Task Force submits an updated ruleset to AG for publication
```

Comment:
Should it be clearer that "Publication" can in itself be a whole process, governed by the AG WG and not just a formal step?

Reply:
I'll put in something to clear that up.